Updated the code example of "How to run Durable functions as WebJobs", for WebJobs SDK 3.x

CarlosSardo opened the pull request:
- Addressing [issue 590](https://github.com/Azure/azure-functions-durable-extension/issues/590): Update the code example of ["How to run Durable functions as WebJobs"](https://docs.microsoft.com/en-us/azure/azure-functions/durable/durable-functions-webjobs-sdk), for WebJobs SDK 3.x.

CarlosSardo committed:
Update durable-functions-webjobs-sdk.md

Updated the code example of "How to run Durable functions as WebJobs" at, https://docs.microsoft.com/en-us/azure/azure-functions/durable/durable-functions-webjobs-sdk, for WebJobs SDK 3.x.

CarlosSardo committed:
Merge pull request #1 from CarlosSardo/CarlosSardo-patch-590

Update durable-functions-webjobs-sdk.md

cgillum commented on articles/azure-functions/durable/durable-functions-webjobs-sdk.md:
```diff
@@ -214,50 +214,63 @@ This section provides an overview of how to run the [sample project](https://git
+            });
+        })
+
+        .UseConsoleLifetime();
```
To make the code easier to comprehend, I suggest making the following whitespace adjustments:

1. Add another level of indenting before each `.Configure...()` and `.Use...()` method calls.
2. Remove the blank lines between each call to `.Configure...` and `.Use...`.

It was actually quite hard for me to understand the code because of the whitespace issues and I think these small improvements will help quite a bit.

CarlosSardo replied:
Will do!

CarlosSardo committed:
Update durable-functions-webjobs-sdk.md

- Added another level of indenting before each .Configure...() and .Use...() method calls.
- Removed the blank lines between each call to .Configure... and .Use....

CarlosSardo committed:
Update durable-functions-webjobs-sdk.md

Removed blank line before .config.AddDurableTask(...
